pkg/stanza/docs/operators/container.md

```diff
@@ -251,4 +251,71 @@ Configuration:
 
 </td>
 </tr>
-</table>
\ No newline at end of file
+</table>
+
+#### Parse multiline logs and recombine into a single one
+
+If you are using the Docker format (or log tag indicators are not working), 
+you can recombine multiline logs into a single one.
+
+Configuration:
+```yaml
+receivers:
+  filelog:
+    include:
+      - /var/log/pods/*/my-service/*.log
+    include_file_path: true       
+    operators:
+    - id: container-parser
+      type: container
+    - id: recombine
+      type: recombine
+      combine_field: body
+      is_first_entry: body matches "^\\d{4}-\\d{2}-\\d{2}T\\d{2}"
+```
+
+<table>
+<tr><td> Input body </td> <td> Output body</td></tr>
+<tr>
+<td>
+
+```json lines
+{
+  "timestamp": "",
+  "body": "{\"log\":\"2024-07-03T13:50:49.526Z  WARN 1 --- [http-nio-8080-exec-6] c.m.antifraud.FraudDetectionController   : checkOrder\",\"stream\":\"stdout\",\"time\":\"2029-03-30T08:31:20.545192187Z\"}",
```

Comment:
nit: the year in the `time` field is not the same as in the parsed `timestamp`:
`2029-03-30T08:31:20.545192187Z` VS `2024-03-30 08:31:20.545192187 +0000 UTC` (2029 vs 2024).

Same for the next log line.

Reply:
good catch